smartdevicelink/sdl_java_suite
# Add Windows Symlink builder script

Fixes #1062 

This PR is **ready** for review.

### Risk
This PR makes **no** API changes.

### Testing Plan
- Open the Android project in Android Studio on a Linux/Mac machine, verify it builds
- Open the Android project in Android Studio on a Windows machine, run the gradle task `buildWindowSymLinks`,  verify it builds

### Summary
- Added a script to build Windows type symlinks
- Added a gradle task to call script
- Added a gradle conditional to point to correct folder based on OS

### Changelog

##### Bug Fixes
* Windows users can now build the project from source if they wish


### CLA
